src/main/java/com/google/api/generator/gapic/composer/defaultvalue/DefaultValueComposer.java

```diff
@@ -168,6 +168,20 @@ public static Expr createValue(
               .setStaticReferenceType(field.type())
               .setMethodName("newBuilder")
               .build();
+      if (field.type().equals(TypeNode.VALUE)) {
```

Comment:
Not sure if I understand why we need to set the default value for response, can you please provide a little more explanation? Also looks like this method is only called by test composers?

Reply:
Honestly, I don't know myself what happens. Looks like a but in json serialization/deserialization for Value class. It is a special class (like `Object` in java or `void *` in C). Due to this when we instantiate `Value.newBuilder.build()` it is recognized as an empty value, but when serialized->deserialized to/from json, then it gets recognized as special `null` value, and as a result the `Assert.equals()` throws.

Reply:
Maybe I'm missing something, what I don't understand is that this code only adds a default boolean value "true" to the expected response in the generated tests, where is this value coming from in the actual response?

Reply:
In each generated test there is `MockHttpService` instance used. Before calling a tested method, the expected response is added to the mockService. The response returned from the tested methos is the expectedResult object which went through serialization/deserialization cycle.